src/ast/query.rs

```diff
@@ -152,6 +153,21 @@ impl fmt::Display for SetQuantifier {
         }
     }
 }
+
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+pub struct Table {
+    pub table_name: String,
+    // pub schema_name: String,
```

Comment:
I was having a bit of trouble parsing something like `my_schema_name.my_table_name`, but I imagine that we'd ideally want the schema name to be a part of the `Table` struct.